src/corehost/cli/deps_entry.cpp

```diff
@@ -104,8 +104,8 @@ bool deps_entry_t::to_full_path(const pal::string_t& base, pal::string_t* str) c
     }
 
     pal::string_t new_base = base;
```

Comment:
Could you please make the change in `fx_muxer.cpp` as well?
https://github.com/dotnet/cli/blob/rel/1.0.0/src/corehost/cli/fxr/fx_muxer.cpp#L130

pal::to_lower starting from the package name onwards?